### ISSUE TITLE
CONTRIBUTING & themes and css updates

### ISSUE THREAD

**Issue by Kernix13** [volca/markdown-preview#138]
I found your chrome extension last night and love it but you only have 1 dark theme and the font-size is too small. So I found the CSS file loading with the dark theme, made some changes (increased font-size, changed margin/padding `px` to `em` or `rem`, ...) and then switched to that custom file. 

Are you open to modifications to your current themes or having new themes? However, on your CONTRIBUTING page, the branch `dev` is mentioned but I only see a `master` branch. I am not a seasoned pro when it comes to contributing, but I think your notes under the heading _Keeping your fork in sync_ are wrong. My notes for staying up to date are:

```
git checkout master
git fetch upstream
git merge upstream/master
```

But you have the fetch command before the checkout to dev/master. I also never did a `git rebase` command before `git push` so I'm confused about that step in your contributing file. If you are interested in some new styles/themes, then I'll fork the repo and get started. Otherwise, I'll just continue to make changes on my end.

**Comment by volca** [comment]
Thanks for note this. I've fixed the CONTRIBUTING page. Welcome for contribute!

**Comment by Kernix13** [comment]
Ok, thanks.

So I prefer a dark theme and the only dark one you have is _ClearnessDark_ but I found the font-size too small so that is the main thing I changed, but I changed a number of other things. I found the CSS file when I inspected the page using that theme. 

I assume you would not want edits to ClearnessDark.css, so should I push a new theme/file called ClearnessDarkLg.css? 

Also, I could look in the code but I found the dark red text very hard to see but I can't seem to change it. I think `css/highlight.css` has higher specificity than anything in my file. Any ideas of why that would be or how to overwrite it? I'll add `!important` on my end and see if that does it.

**Comment by Kernix13** [comment]
Ok, I forked and cloned your repo and took a look at the files. How much should I do? I see the `*.css` files in the `themes` folder so I can add my file `ClearnessDarkLg.css` to that folder. But I also noticed that the themes are referenced in an array named `defaultThemes` in `js/options.js`, `js/markdownify.js`, and `js/popup.js`, and they are in option tags in `options.html` and `popup.html`. I assume I should add `ClearnessDarkLg` to those files as well. Or do you want to do that? 

I'm not sure if you want to check the CSS and see how it looks before you add it to the extension. I assume once it is in the 2 HTML files, then users will see it. 

I'll go ahead and make all the changes but I'll wait to push the files until I hear back from you. 

FYI, I like all my CSS styles except for the paragraph `code` tag styling. I would love to hear your thoughts about that.

**Comment by volca** [comment]
You're right. It's a little complex to add a new theme css file. I modified the [js/config.js](https://github.com/volca/markdown-preview/blob/4f053b1708f86df9c7e6d2b341537f499c3d5dc0/js/config.js#L20). 

Here're the steps for add new theme

* Append your theme to `themes` in `js/config.js`
* Add your theme css file in the `themes` folder

**Comment by Kernix13** [comment]
Ok, I just fetched your changes and will do that. However, I'm getting `package.lock.json` as a new untracked file as a result of running `npm install`. I can either 

1. push that file and you can keep it or delete it, or 
2. I can add that file to `.gitignore` then that will will show as changed, or 
3. I can delete it on my end

Let me know which you prefer.

**Comment by Kernix13** [comment]
Also, I think you need the theme options back in `options.html` because now there is nothing to select.

**Comment by volca** [comment]
You don't need run the `npm install` because I've copied all necessary js files to the `js` folder.  But I just updated [.gitignore](https://github.com/volca/markdown-preview/blob/master/.gitignore#L6) file

The options.html works well at my end. Please check [my archived extension file](https://github.com/volca/markdown-preview/files/10809527/markdown_preview.zip). You can check the error log with chrome's console

**Comment by Kernix13** [comment]
Ok, I'll fetch the `.gitignore` update, add the CSS and the theme in `config.js` and then push.

**Comment by Kernix13** [comment]
Forget that, It's late and I'm tired and I just added the css file on master instead of on a new branch. Let me do it in the morning after I had my coffee. I don't want to mess anything up

**Comment by Kernix13** [comment]
I told you I was a newbie at this stuff. I think I pushed an empty `ClearnessDarkLg.css` file. Or at least I did not see any styles in it when I looked on my branch `new-theme` in my account. So I noticed I did not save the file, so I pushed it again after saving but now I do not see the _Compare & Pull Request_ button on my end. Let me know what you see on your end.

**Comment by volca** [comment]
Closed
